src/main/webapp/resources/search/js/search.js

```
@@ -40,96 +40,70 @@ const state = {
+            document.dispatchEvent(new CustomEvent("tripTypeChanged"));
         }
-    }
 
-    document.dispatchEvent(new CustomEvent("tripTypeChanged"));
+        // active 클래스 토글
+        tabs.forEach(t => t.classList.toggle("search-tab--active", t === btn));
+
+        // UI 반영
+        syncTripUI();
+        syncDateSummary();
+    });
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)